regolith/helpers/makeappointmentshelper.py

```diff
@@ -88,40 +95,61 @@ def sout(self):
         gaps = []
         outdated = []
         depleted = []
-
-        if rc.begin_date and rc.date:
-            raise ValueError("Please enter either an interval or a date, not both")
-        if rc.date:
+        if rc.underspent:
+            underspent =[]
+        if rc.overspent:
+            overspent = []
+
+        if rc.begin_date:
+            begin_date = date_parser.parse(rc.begin_date).date()
+            if rc.desired_date:
+                raise ValueError("Please enter either an interval or a date, not both")
+            if not rc.end_date:
+                end_date = begin_date + relativedelta(years=1)
+        elif rc.desired_date:
             desired_date = date_parser.parse(rc.date).date()
         else:
             desired_date = dt.date.today()
+
         for person in self.gtx["people"]:
             appts = person.get("appointments")
-            if rc.gap:
-                if rc.begin_date:
-                    if not is_fully_appointed(appts, begin=rc.begin_date, end=rc.end_date)[0]:
-                        gaps.append(person.get('name'))
-                        gaps.append((is_fully_appointed(appts, begin=rc.begin_date, end=rc.end_date))[1])
-                else:
-                    if not is_fully_appointed(appts, now=desired_date)[0]:
-                        gaps.append(person.get('name'))
-                        gaps.append((is_fully_appointed(appts, now=desired_date))[1])
-            if rc.out_of_date:
+            #if we are checking over an interval of time
```

Comment:
I think we are always doing this.

Reply:
I'm not sure why this is happening – I removed this from the code after your last review, in the commit `removed single day functionality` above –  maybe GitHub is not refreshing the files?

Reply:
first, make sure that you pushed, then on GH click "files changed" to get the most recent versions.  I may have made a mistake here too....